test/src/test/java/org/apache/accumulo/test/util/CertUtilsTest.java

```diff
@@ -164,25 +172,27 @@ public void signingChain() throws Exception {
     // no reason the keypair we generate for the tservers need to be able to sign anything,
     // but this is a way to make sure the private and public keys created actually correspond.
     CertUtils certUtils = getUtils();
-    File tempSubDir = tempDir.toPath().resolve(testName()).toFile();
-    assertTrue(tempSubDir.isDirectory() || tempSubDir.mkdir());
-    File rootKeyStoreFile = tempSubDir.toPath().resolve("root.jks").toFile();
-    certUtils.createSelfSignedCert(rootKeyStoreFile, "test", PASSWORD);
-    File signedCaKeyStoreFile = tempSubDir.toPath().resolve("signedca.jks").toFile();
-    certUtils.createSignedCert(signedCaKeyStoreFile, "test", PASSWORD,
-        rootKeyStoreFile.getAbsolutePath(), PASSWORD);
-    File signedLeafKeyStoreFile = tempSubDir.toPath().resolve("signedleaf.jks").toFile();
-    certUtils.createSignedCert(signedLeafKeyStoreFile, "test", PASSWORD,
-        signedCaKeyStoreFile.getAbsolutePath(), PASSWORD);
+    Path tempSubDir = tempDir.resolve(testName());
+    if (!Files.isDirectory(tempSubDir)) {
+      Files.createDirectories(tempSubDir);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier